src/VCR/Storage/AbstractStorage.php

```diff
@@ -69,14 +71,15 @@ public function __construct($cassettePath, $cassetteName, $defaultContent = '[]'
 
         if (!file_exists($file) || 0 === filesize($file)) {
             file_put_contents($file, $defaultContent);
```

Comment:
i don't check for $writeable in this case. this is to keep strict BC - if the file is missing and we provide a $defaultContent or there are no requests, there is no error.

on a readonly filesystem, the file_put_contents will fail however. but i guess we don't care about that, its not different than before.